README.md

```diff
@@ -1,72 +1,72 @@
-We encourage you to append implementation suggestions as this helps to decrease the
-workload of the very limited maintenance team. 
+| Directory/File        | Content                                                   |
+| --------------------- | --------------------------------------------------------- |
+| `./Boards/`           | Driver Validation examples for various boards             |
+| `./Config/`           | Configuration files for the driver validation framework   |
+| `./Documentation/`    | Placeholder for the offline documentation in the pack     |
+| `./DoxyGen/`          | Source of the documentation                               |
+| `./Include/`          | Header files for driver validation components             |
+| `./Scripts/`          | Script files for XML reports                              |
+| `./Source/`           | Source files for driver validation components             |
+| `./Tools/`            | Server side tools for testing drivers                     |
+| `./ARM.CMSIS-Driver_Validation.pdsc` | Open-CMSIS-Pack description file           |
+| `./gen_pack.sh`       | Open-CMSIS-Pack generation script                         |
+| `./LICENSE.txt`       | License text for the repository content                   |
 
-We will be monitoring and responding to issues as best we can.
-Please attempt to avoid filing duplicates of open or closed items when possible.
-In the spirit of openness we will be tagging issues with the following:
 
-- **bug** – We consider this issue to be a bug that will be investigated.
 
-- **wontfix** - We appreciate this issue but decided not to change the current behavior.
+## Examples
 
-- **enhancement** – Denotes something that will be implemented soon. 
+Folder [./Boards/](./Boards/) contains example projects that show how to use the CMSIS-Driver Validation on a real hardware with available CMSIS-Driver implementations.The examples are also included in CMSIS-Driver Validation Software Pack. Details are explained in [Examples documentation](https://arm-software.github.io/CMSIS-Driver_Validation/main/examples.html).
 
-- **future** - Denotes something not yet schedule for implementation.
+## Build CMSIS-Driver Validation as Open-CMSIS-Pack
 
-- **out-of-scope** - We consider this issue loosely related to CMSIS. It might by implemented outside of CMSIS. Let us know about your work.
-
-- **question** – We have further questions to this issue. Please review and provide feedback.
-
-- **documentation** - This issue is a documentation flaw that will be improved in future.
-
-- **review** - This issue is under review. Please be patient.
+ A generator script `./gen_pack.sh` based on [gen-pack utility](https://github.com/Open-CMSIS-Pack/gen-pack) is provided for building the CMSIS-Driver Validation as [Open-CMSIS-Pack](https://open-cmsis-pack.github.io/Open-CMSIS-Pack-Spec/main/html/index.html). Just follow the steps below:
```

Comment:
I would change './gen_pack.sh" to just 'gen_pack.sh'
I would change "gen-pack utility" to "gen-pack library" since that is actually gen-pack library which contains multiple utilities

Reply:
Agree, will update.